metasv/run_spades.py

```diff
@@ -171,13 +184,22 @@ def run_spades_parallel(bam=None, spades=None, bed=None, work=None, pad=SPADES_P
     chrs = set(chrs)
     all_intervals = [interval for interval in bedtool] if not chrs else [interval for interval in bedtool if
                                                                          interval.chrom in chrs]
+<<<<<<< HEAD
```

Comment:
Looks like the merge conflict was not resolved properly. You will need to keep one of these <<<, ===, >>> blocks after fixing the right now. Right now, this shows a merge conflict.